docs/get-started/sumo-logic-ui-new.md

```diff
@@ -211,6 +211,85 @@ You don't have to stop there either. You can take the next step and become Sumo
+
+Having in-app tabs has caused performance issues, which has been a complaint about the Sumo Logic UI. On one hand, consolidating all the tabs together reduces the number of browser tabs, but on the other hand, it can make it difficult to manage multiple sessions.
+
+If you prefer having all Sumo Logic tabs in one place - without getting mixed with other browser tabs - we suggest utilizing the [tab grouping functionality](#customize-your-environment-withtabs) offered by most browsers. By adding your Sumo Logic tab to a tab group, any new tabs opened within the Sumo Logic platform will automatically open in the same tab group. You can also collapse the tab group to reclaim valuable real estate in the browser's tab bar.
+
+### The old (Classic) UI remembers the tabs that I had opened in my previous session. Will this functionality be in the New UI?
+
+After analyzing tab usage data, we've found that a minimal portion of previously opened tabs are actively utilized by our users. Consequently, the Sumo Logic UI often remains cluttered with multiple unused tabs. With the introduction of the New UI experience, if you fail to close browser tabs from previous sessions, they will automatically reload upon login.
+
+<img src={useBaseUrl('img/get-started/tab-reload.gif')} alt="tab-reload.gif" />
+
+Moreover, we've made the **Recents** feature more prominent in the navigation bar and plan to extend it to other content types which will make it easier for users to open recently opened tabs.
+
+### If I opt in to the New UI, will I be locked out of the old (Classic) UI?
+
+No. You will have the capability of opting in and out of the New UI experience from within the UI.  
+
+When the feature flag is enabled, all users within the organization will have access to a **Switch to New UI** on the left navigation menu. By clicking on this button, users can opt-in to the New UI experience. Once opted in, the New UI will be remembered for the user, resulting in the rerouting of old URLs to new ones. Consequently, when users log back in, they will be directed to the New UI automatically.
+
+### How do I opt out of the New UI?
+
+You have the flexibility to opt out whenever you choose. Just click the **Return to Classic UI** option on your left navigation menu of the New UI. Once you've switched back to the Classic UI, all subsequent sessions will default to the Classic UI experience.
+
+If you enter the New UI URL in your browser, it will still direct you to the New UI. We have implemented a translation from the old (Classic) UI to the New UI, but not the other way around.
+
+### When is Sumo Logic planning to remove access to the Classic UI?
+
+Our goal is to turn off access to the old (Classic) UI at the end of 2024. However, this timeline is subject to our successful Beta testing.
+
+### Are there certain capabilities that are not going to be available during Beta?
```

Comment:
@kimsauce we might have to make some additions to this section. But it's good for first round.

Reply:
Sounds good!